extensions/sql-database-projects/src/test/projectController.test.ts

```diff
@@ -166,7 +166,7 @@ describe.skip('ProjectsController: project controller operations', function ():
 			let profilePath = await testUtils.createTestFile(baselines.publishProfileBaseline, 'publishProfile.publish.xml');
 			const projController = new ProjectsController(testContext.apiWrapper.object, new SqlDatabaseProjectTreeViewProvider());
 
-			let result = await projController.readPublishProfile(vscode.Uri.parse(profilePath));
+			let result = await projController.readPublishProfile(vscode.Uri.file(profilePath));
```

Comment:
is it better to use Uri.file() over Uri.parse()?

Reply:
For file paths, yeah - it accounts for Windows- and Unix-style paths together, while `.parse()` expects things to all use forward slashes (better for treeItem URIs or web URLs... or anything other than Windows filesystem paths, lol).

As is, the test in a slight corner-case: my repo is on `F:\`, but the test files get created in the user temp folder, which is on `C:\`.  When the URI gets read with `.parse()`, it ends up looking in `F:\path\to\temp\profile.xml` instead of `C:\path\to\temp\profile.xml`